config-schema.json

```diff
@@ -151,12 +151,22 @@
         "usage_data": {
             "type": "object",
             "additionalProperties": false,
+            "collection_enabled": {
+                "type": "boolean"
+            },
             "properties": {
                 "gateway": {
                     "type": "object",
                     "additionalProperties": false,
                     "properties": {
                         "dispatch_enabled": {
+                            "type": "boolean",
+                            "deprecationMessage": "The `usage_data.gateway.dispatch_enabled` option is deprecated and will be removed in v6.8.0 Please use the `usage_data.gateway.entity_dispatch_enabled` and `usage_data.gateway.consent_dispatch_enabled` option instead."
+                        },
+                        "entity_dispatch_enabled": {
+                            "type": "boolean"
+                        },
+                        "consent_dispatch_enabled": {
                             "type": "boolean"
                         },
```

Comment:
We decided to remove that these two new configs and always dispatch the consent. We will not deprecate `dispatch_enabled` and just introduce `collection_enabled` to be able to disable the whole process (and not only the part where we would actually dispatch entities) :smile: